src/providers/historical-telemetry-provider.js

```diff
@@ -116,6 +116,8 @@ export default class YamcsHistoricalTelemetryProvider {
                     value: sample.min,
                     id: id
                 };
+
+                addLimitInformation(sample, min_value);
```

Comment:
From looking at the API it appears that samples don't provide limit data (https://docs.yamcs.org/yamcs-http-api/parameter-archive/get-parameter-samples/). Can you confirm whether or not you were seeing limits for samples in your testing?

If samples don't return information on limit violations we're in trouble. This means our plots can't show historical limit violations.

This...is a problem because it means that our plots can't show historical limit violations.

Reply:
hmm you are correct, looks like this is an issue. 
eg:  response for following,
http://localhost:9000/yamcs-proxy/api/archive/myproject/parameters/Demo/ccsdsPacketVersion/samples?start=2021-02-01T18:25:10.670Z&stop=2021-02-10T18:55:10.670Z&count=207&order=asc
```
{
  "sample": [{
    "time": "2021-02-10T17:52:25.372Z",
    "avg": 0.0,
    "min": 0.0,
    "max": 0.0,
    "n": 2745
  }, {
    "time": "2021-02-10T18:55:10.589Z",
    "n": 0
  }]
}
```